src/raftstore/store/store.rs

```diff
@@ -1828,55 +1828,99 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         self.register_compact_check_tick(event_loop);
     }
 
-    fn on_split_check_result(
+    fn on_prepare_split_region(
         &mut self,
         region_id: u64,
         epoch: metapb::RegionEpoch,
-        split_key: Vec<u8>,
+        split_key: Vec<u8>, // `split_key` is a encoded key.
+        cb: Option<Callback>,
```

Comment:
Can `cb` be handled outside the method? So there won't be so many places that need to expand cb.